src/test/ui/generic-associated-types/issue-47206-where-clause.stderr

```diff
@@ -1,11 +1,11 @@
 error[E0276]: impl has stricter requirements than trait
-  --> $DIR/issue-47206-where-clause.rs:13:5
+  --> $DIR/issue-47206-where-clause.rs:13:40
    |
 LL |     type Assoc3<T>;
    |     --------------- definition of `Assoc3` from trait
 ...
 LL |     type Assoc3<T> where T: Iterator = Vec<T>;
-   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ impl has extra requirement `T: Iterator`
+   |                                        ^^^^^^ impl has extra requirement `T: Iterator`
```

Comment:
This has an incorrect span, this should point at either the whole assoc type _or_, even better yet, at the `T: Iterator` extra requirement. For that last part we'd need specific logic doing that.